### PR TITLE
Add django-sanitized-dump for making an anonymised database dump

### PR DESCRIPTION
**Pull request by mikkokeskinen**
See [docs/database_dump.adoc](https://github.com/City-of-Helsinki/open-city-profile/blob/059e87038aa90c8c0c5cd6bca6c5021059526b4a/docs/database_dump.adoc) for more information.

Refs HP-808

### CODE REVIEW

File: .sanitizerconfig [hunk 1]
```diff
@@ -0,0 +1,417 @@
...
+    national_identification_number: "string.empty"
+    profile_id: null
+  profiles_verifiedpersonalinformationpermanentaddress:
+    id: null
+    post_office: "profile.encrypted_city"
+    postal_code: "profile.encrypted_postal_code"
+    street_address: "profile.encrypted_street_address"
+    verified_personal_information_id: null
+  profiles_verifiedpersonalinformationpermanentforeignaddress:
+    additional_address: null
```
Comment (akikoskinen) [inline .sanitizerconfig:265]: `profiles_verifiedpersonalinformationpermanentforeignaddress.additional_address` should probably be sanitised?

File: sanitizers/profile.py [hunk 2]
```diff
@@ -0,0 +1,93 @@
...
+
+def sanitize_encrypted_city(value):
+    return as_encrypted_hex_string(fake.city())
+
+
+def sanitize_city(value):
+    return fake.city()
+
+
+def sanitize_encrypted_city_number(value):
```
Comment (akikoskinen) [inline sanitizers/profile.py:52]: Maybe this could be called `sanitize_encrypted_municipality_number`.

**Review by akikoskinen — COMMENTED** [review]
A few noticed points, nothing much.

**Comment by mikkokeskinen** [comment]
Good points. Changed those.

**Review by akikoskinen — APPROVED** [review]

**Review by tituomin — APPROVED** [review]

**Pull request merged**